experimental/framework/tree-react-api/src/test/reactSharedTreeView.spec.tsx

```diff
@@ -41,4 +45,36 @@ describe("reactSharedTreeView", () => {
 		dataObject.treeView.root.nuts += 1;
 		assert.equal(dataObject.treeView.root.bolts, 6);
 	});
+
+	describe("dom tests", () => {
+		let cleanup: () => void;
+
+		before(() => {
+			cleanup = globalJsdom();
+		});
+
+		after(() => {
+			cleanup();
+		});
+
+		for (const reactStrictMode of [false, true]) {
+			describe(`StrictMode: ${reactStrictMode}`, () => {
+				const builder = new SchemaFactory("tree-react-api");
+
+				class Item extends builder.object("Item", {}) {}
+
+				const View = ({ root }: { root: Item }): React.JSX.Element => <span>View</span>;
+
+				it("TreeViewComponent", () => {
+					const view = independentView(new TreeViewConfiguration({ schema: Item }), {});
+					const content = <TreeViewComponent viewComponent={View} tree={{ treeView: view }} />;
+					const rendered = render(content, { reactStrictMode });
+					assert.match(rendered.baseElement.textContent ?? "", /Document is incompatible/);
```

Comment:
Nit: would probably be good to add a couple of quick comments explaining the text checks.